addons/survey/models/survey_survey.py

```diff
@@ -349,6 +349,29 @@ def write(self, vals):
             return self.sudo()._handle_certification_badges(vals)
         return result
 
+    @api.returns('self', lambda value: value.id)
+    def copy(self, default=None):
+        """ Correctly copy the 'triggering_question_id' and 'triggering_answer_id' fields from the original
+        to the clone.
+        This needs to be done in post-processing to make sure we get references to the newly created
+        answers/questions from the copy instead of references to the answers/questions of the original.
+        This implementation assumes that the order of created questions/answers will be kept between
+        the original and the clone, using 'zip()' to match the records between the two.
+        """
+        self.ensure_one()
+        clone = super(Survey, self).copy(default)
```

Comment:
What happens if I set a default on question_ids ?

Reply:
Sorry for the delay, yes indeed it could desynchronize the copy (and would not even be a real copy). 
As I suppose there is no use-case for that, I have added a check at the beginning that raises a validation error in that case.
Do we need to translate such internal error ?